othello/apps/tournaments/forms.py

```diff
@@ -10,6 +11,7 @@
 
 
 class TournamentCreateForm(forms.ModelForm):
+    qs = Submission.objects.latest().exclude(user=get_user_model().objects.get(username="Yourself"))
```

Comment:
Who creates the `Yourself` user? (i.e. should we do a `get_or_create` here?) I'm just not sure, just trying to find what could go wrong

Reply:
The "Yourself" `User` and `Submission` are created by a [data migration](https://docs.djangoproject.com/en/3.2/topics/migrations/#data-migrations) in [this migration](https://github.com/tjcsl/othello-tourney/blob/master/othello/apps/games/migrations/0020_auto_20200617_2253.py)